### PR TITLE
#3744 - Profitwell new actions

### LINKED ISSUE #3744: ProfitWell
### App
ProfitWell

### Actions
* Add Customer
* Churn Subscription
* Get Customer Info
* Upgrade/Downgrade Subscription

### PR DESCRIPTION
**Pull request by GTFalcao**
# New Actions
- **Create Subscription** _(add customer)_
- **Churn Subscription**
- **Get Customer Info**
- **Update Subscription** _(upgrade/downgrade subscription)_

Closes #3744 

<a href="https://gitpod.io/#https://github.com/PipedreamHQ/pipedream/pull/4290"></a>

### CODE REVIEW

**Comment by dylburger** [comment]
Thanks for submitting this PR! When we review PRs, we follow the [Pipedream component guidelines](https://pipedream.com/docs/components/guidelines/). If you're not familiar, here's a quick checklist:

- [ ] [Create components to address specific use cases whenever possible](https://pipedream.com/docs/components/guidelines/#component-scope)
- [ ] [Component `key`s should follow the format `app_name_slug-slugified-component-name`](https://pipedream.com/docs/components/guidelines/#component-key-pattern)
- [ ] Components should follow the [standard directory structure](https://pipedream.com/docs/components/guidelines/#folder-structure)
- [ ] [Prefer Node.js client libraries to REST APIs](https://pipedream.com/docs/components/guidelines/#using-apis-vs-client-libraries)
- [ ] When making API requests, [handle pagination](https://pipedream.com/docs/components/guidelines/#pagination) to ensure all data / events are processed
- [ ] [Use `secret` props](https://pipedream.com/docs/components/guidelines/#capturing-sensitive-data) to capture sensitive data
- [ ] Props and methods should be defined in [app files](https://pipedream.com/docs/components/guidelines/#app-files) whenever possible
- [ ] Document methods with [JS Docs](https://pipedream.com/docs/components/guidelines/#methods)
- [ ] [Use `optional` props](https://pipedream.com/docs/components/guidelines/#optional-vs-required-props) whenever possible, and [set a `default` value](https://pipedream.com/docs/components/guidelines/#default-values) where you can
- [ ] Use [async options](https://pipedream.com/docs/components/guidelines/#async-options) to accept user input wherever possible

**Review by andrewjschuang — COMMENTED** [review]
Hey @GTFalcao, just a minor comment. LGTM!

File: components/profitwell/package.json [hunk 1]
```diff
@@ -15,5 +15,11 @@
   "license": "MIT",
   "publishConfig": {
     "access": "public"
+  },
+  "dependencies": {
+    "@pipedream/platform": "^1.1.1"
+  },
+  "devDependencies": {
+    "@pipedream/types": "^0.1.4"
```
[the remark this answers is not in the document]
Comment (GTFalcao) [reply components/profitwell/package.json:23]: Hey, I thought it was best this way as it's only used for the Typescript dev environment, while at runtime the output .mjs files are used - does that make sense? @andrewjschuang

[the remark this answers is not in the document]
Comment (andrewjschuang) [reply components/profitwell/package.json:23]: Makes sense! Although not sure if it makes a difference in Pipedream runtime

**Review by andrewjschuang — APPROVED** [review]

**Pull request merged**